server.js

```diff
@@ -236,7 +235,8 @@ function cache(f) {
 
     // Only call vendor servers when last request is older than…
     var cacheInterval = 5000;  // milliseconds
-    var cachedRequest = function (uri, options, callback) {
+    var cachingRequest = function (uri, options, callback) {
+      var request = require('request');
```

Comment:
Why do you want to require request every time we need it, instead of once and for all?

If you're worried about mistakenly calling it, we can change its name to `httpRequest` or something like that.